### ISSUE TITLE
[MacOS] PlatformExeption: Failed to aquire root FlutterViewController

### ISSUE THREAD

**Issue by edeuss** [ThexXTURBOXx/flutter_web_auth_2#10]
## Describe the bug
Getting a PlatformExeption from running FlutterWebAuth2.authenticate on a mac.

## Device (please complete the following information!)
 - Device: Macbook Pro
 - OS: MacOS V12.6
 - `flutter_web_auth_2` version: 1.1.2

## Code
```
try {
      final result = await FlutterWebAuth2.authenticate(
        url:
            "https://example.com",
        callbackUrlScheme: "example",
      );
} catch (error) {
      print(error);
}
```

**Comment by ThexXTURBOXx** [comment]
The full error message is missing.
Also, please try to apply the fixed Andrei and I noted in #9

**Comment by ThexXTURBOXx** [comment]
I tagged this PR as being part of `Hacktoberfest`.
So, if someone is able to fix and properly test this issue, he is one step closer to completing `Hacktoberfest 2022` :)

**Comment by Adam-Langley** [comment]
Also fixed by e89919253da91f3e860f23f8e545d33a66b455e3

**Comment by ThexXTURBOXx** [comment]
@Adam-Langley Thanks for letting me know! In case someone has more problems with this, I can reopen the issue!
